packages/components/scripts/vendorism/custom/tooltip.vendorism.js

```diff
@@ -10,11 +12,19 @@ const FILES_TO_TRANSFORM = [
  */
 const transformComponent = (path, originalContent) => {
   // Get a 4px spacing between nupsi and anchor
-  const content = originalContent.replace(
+  let content = originalContent.replace(
     '@property({ type: Number }) distance = 8;',
     '@property({ type: Number }) distance = 13;',
   );
 
+  // #849: Mark hoist as deprecated
+  content = addSectionAfter(
+    originalContent,
```

Comment:
The `content` needs to be used there instead of the `originalContent`. Otherwise the first change with the new distance value is not taking action.

```suggestion
    content,
```